solarsystem/style.css

```diff
@@ -0,0 +1,299 @@
+body {
+    margin: auto;
+    background-image:url(./image/space.jpg);
+    background-position: center;
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+    
+}
+
+#solarsystem{
+    position: relative;
+    margin: auto;
+    margin-top: 100px;
+    width: 900px;
+    height:900px;
+    padding:50px
+
+}
+
+  #neptune{
+position: absolute;
+width: 900px;
+height: 900px;
+background-color: rgba(0, 0, 0, 0);
+border: 2px dotted #929090;
+border-radius: 100%;
+left: -40px;
+top: -40px;
+animation: solar 120s linear infinite;
+}
+
+#neptune::after{
+position: absolute;
+width: 45px;
+height: 45px;
+border-color: #00009d;
+border-radius: 100%;
+background-image:url(./image/neptune.jpg);
+background-position: center;
+box-shadow:  rgb(15, 15, 13) 0px 0px 5px 1px inset ;
+content:'';
+left: 300px;
+top: -13px;
+} 
+
+#uranus{
+    position: absolute;
+    width: 820px;
+    height: 820px;
+    background-color: rgba(0, 0, 0, 0);
+    border: 2px dotted #929090;
+    border-radius: 100%;
+    left: 0px;
+    top: 0px;
+    animation: solar 160s linear infinite;
+}
+    
+#uranus::after{
+    position: absolute;
+    width: 35px;
+    height: 35px;
+    border-radius: 100%;
+    background-image:url(./image/uranus.jpeg);
+    background-position: center;
+    box-shadow:  rgb(15, 15, 13) 0px 0px 5px 1px inset ;
+    content:'';
+    left: 295px;
+    top: -10px; 
+}
+
+ 
+#saturn{
+    position: absolute;
+    width: 750px;
+    height: 750px;
+    background-color: rgba(0, 0, 0, 0);
+    border: 2px dotted #929090;
+    border-radius: 100%;
+    left: 30px;
+    top: 30px;
+    animation: solar 70s linear infinite;
+}
+#saturn::before{
+    position: absolute;
+    width: 55px;
+    height: 55px;
+    border-radius: 100%;
+    background-image:url(./image/saturn.jpg);
+    background-position: center;
+    box-shadow:  rgb(15, 15, 13) 0px 0px 5px 1px inset ;
+    content: '';
+    left: 230px;
+    top: -11px;   
+}
+
+#saturn::after{
+    position: absolute;
+    width: 65px;
+    height: 20px;
+    border-radius: 100%;
+    background-color: rgba(0, 0, 0, 0);
+    border-color: rgb(231, 215, 197);
+    border-style: solid;
+    border-width: 0px 4px 8px 4px;
+    content: '';
+    left: 224px;
+    top: 3px;
+   
+}
+
+#jupiter{
+    position: absolute;
```

Comment:
It would be good to use class composition to avoid repeating common styles

```

.planet-orbit {
  position: absolute;
  background-color: rgba(0, 0, 0, 0);
  border: 2px dotted #929090;
  border-radius: 50%;
}

#jupiter-orbit {
  Some specific styling
}

```